From fbbcb0c03f0528d36ece5b4868112d36f7256aa5 Mon Sep 17 00:00:00 2001
From: Benjamin Berg <bberg@redhat.com>
Date: Thu, 2 Sep 2021 15:55:43 +0200
Subject: [PATCH 06/86] build: Remove "deprecated" option

The functionality that this option was disabling/enabling has been
removed.
---
 configure.ac | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8531137..300e840 100644
--- a/configure.ac
+++ b/configure.ac
@@ -61,15 +61,6 @@ AC_SYS_LARGEFILE
 # Initialize libtool
 LT_INIT
 
-# Build deprecated functionality
-AC_ARG_ENABLE(deprecated, [  --enable-deprecated      build deprecated functionality],
-	      enable_deprecated=$enableval,
-	      enable_deprecated=no)
-if test "${enable_deprecated}" != no; then
-  AC_DEFINE(ENABLE_DEPRECATED, 1, [Define if we should build deprecated functionalty])
-fi
-AM_CONDITIONAL(ENABLE_DEPRECATED, test x$enable_deprecated = xyes)
-
 # Build man pages?
 AC_ARG_ENABLE(man-pages,        [  --enable-man-pages      build manual pages],enable_man_pages=$enableval,enable_man_pages=yes)
 if test "${enable_man_page}" != no; then
-- 
2.35.1

